data/students.json

```diff
@@ -1,75 +1,314 @@
+      "thesisPath": "15kelly/"
     },
     {
-      "displayName": "Kobabe ...",
+      "displayName": "Maia Kobabe",
       "slug": "kobabe",
-      "year": "2015",
-      "thesisPages": 30,
-      "spread": true,
-      "thesisTitle": "...title...",
-      "digitalTitle": "....title...",
-      "digitalUrl": "15guillory/guillory.html"
+      "year": 2015,
+      "thesisPages": 20,
+      "thesisTitle": "The Thief's Tale",
+      "digitalTitle": "Night Driving",
+      "digitalUrl": "15kobabe/kobabe.html",
+      "thesisPath": "15kobabe/"
     },
     {
-      "displayName": "kelly jones",
-      "slug": "kellyjones",
-      "year": "2016",
-      "thesisPages": 8,
-      "spread": true,
-      "thesisTitle": "Shorts",
-      "digitalTitle": "How To Catch a Spider",
-      "digitalUrl": "15guillory/guillory.html"
+      "displayName": "Jacob Magraw",
+      "slug": "magraw",
+      "year": 2015,
+      "thesisPages": 17,
+      "thesisTitle": "Lost Puppet Show",
+      "digitalTitle": "The Base",
+      "digitalUrl": "15magraw/magraw.html",
+      "thesisPath": "15magraw/"
     },
     {
-      "displayName": "Kobabe ...",
-      "slug": "kobabe",
-      "year": "2016",
-      "thesisPages": 30,
-      "spread": true,
-      "thesisTitle": "...title...",
-      "digitalTitle": "....title...",
-      "digitalUrl": "15guillory/guillory.html"
+      "displayName": "Samuel Sattin",
+      "slug": "sattin",
+      "year": 2015,
+      "thesisPages": 21,
+      "thesisTitle": "Dad Sleeps Armageddon",
+      "digitalTitle": "Loon",
+      "digitalUrl": "15sattin/sattin.html",
+      "thesisPath": "15sattin/"
     },
     {
-      "displayName": "kelly jones",
-      "slug": "kellyjones",
-      "year": "2017",
-      "thesisPages": 8,
-      "spread": true,
-      "thesisTitle": "Shorts",
-      "digitalTitle": "How To Catch a Spider",
-      "digitalUrl": "15guillory/guillory.html"
+      "displayName": "Nancy Teresa",
+      "slug": "teresa",
+      "year": 2015,
+      "thesisPages": 15,
+      "thesisTitle": "Fushicho (Wesley)",
+      "digitalTitle": "Gratitude",
+      "digitalUrl": "15teresa/teresa.html",
+      "thesisPath": "15teresa/"
     },
     {
-      "displayName": "Kobabe ...",
-      "slug": "kobabe",
-      "year": "2017",
-      "thesisPages": 30,
-      "spread": true,
-      "thesisTitle": "...title...",
-      "digitalTitle": "....title...",
-      "digitalUrl": "15guillory/guillory.html"
+      "displayName": "Josh White",
+      "slug": "white",
+      "year": 2015,
+      "thesisPages": 20,
+      "thesisTitle": "Bad Year",
+      "digitalTitle": "I Went on a Walk and I Saw a Snake",
+      "digitalUrl": "15white/white.html",
+      "thesisPath": "15white/"
+    },
+    {
+      "displayName": "Shane McDermott",
+      "slug": "mcdermott",
+      "year": 2015,
+      "thesisPages": 14,
+      "thesisTitle": "The Seahorse Adventures",
+      "digitalTitle": "",
+      "digitalUrl": "",
```

Comment:
no data here

Reply:
yeah so there are couple people who don't have digital projects and i think they're just going to be last in the lineup for their year and i'll write some logic to not display anything if there's no data